### DISCUSSION TITLE
Suggestion not provided from time to time after a small glitch thingy.

### DISCUSSION THREAD

**Discussion by faraznisar** [community/community#7254] [Copilot Conversations] [answered]
_Just focus on the bold text and not the rest_

I couldn't find an existing discussion so I thought of creating a new one.

Since copilot works totally different compared to all the other products out there, it is a lot more fun to use and does not annoy me like some other ai systems.

**My concern is that I get a suggestion when I am typing something and before I look at it, the suggestion just disappears (mostly because I type more content and that's my best guess). Is this something that happens with others as well and something GitHub is working on?**

Although, I totally agree that this is one of the best ai out there I have used yet.

**Comment by jansorg** [comment, marked as answer]
You can use shortcut `Alt + \` to manually request completions. This should work in both VSCode and JetBrains IDEs. Pressing Escape will hide the suggestions again.

**Reply by faraznisar** [reply]
Thank you so much. I'm just so used to the ctrl + space that I didn't even bother looking for an alternative. This is cool too though. Hope the support stays this good in the future too though.
